libultraship/libultraship/ImGuiImpl.cpp

```diff
@@ -1522,7 +1615,7 @@ namespace SohImGui {
 
                     if (!isBetaQuestEnabled) {
                         ImGui::PopItemFlag();
-                        ImGui::PopStyleVar();
+                        ImGui::PopStyleVar(1);
```

Comment:
what does adding the `1` here do?

Reply:
If we ever apply a StyleVar in a more global fashion, StyleVars need to be popped properly one by one. Not specifying the "1" here (which is the count of the amount of StyleVars popped), it would incorrectly pop the globally applied StyleVars as well. In this case, it's popping the StyleVar from this:

```
if (!isBetaQuestEnabled) {
                        ImGui::PushItemFlag(ImGuiItemFlags_Disabled, true);
                        ImGui::PushStyleVar(ImGuiStyleVar_Alpha, ImGui::GetStyle().Alpha * 0.5f);
                    }
```

Reply:
Sounds good